bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -37,7 +37,7 @@ <h1 itemprop="name" class="title-shadow-box">Client Bug Bounty Program</h1>
 
       <p>Mozilla will pay a bounty for client and security bugs as detailed below. All security bugs must follow the following general criteria to be eligible:</p>
 
-       <p> Eligible security bugs may be present in any of the current main development or released versions of Firefox, Thunderbird, Firefox for Android, or FirefoxOS as released by Mozilla Corporation (e.g. Nightly mozilla-central, Firefox Developer Edition, or Beta test versions, as well as the final release product versions)</p>
+       <p> Eligible security bugs may be present in any of the current main development or released versions of Firefox or Firefox for Android as released by Mozilla Corporation (e.g. Nightly mozilla-central or Beta test versions, as well as the final release product versions)</p>
```

Comment:
As above, I'd like to clean up the client stuff in a different commit, unless it's really necessary right now.